float64 score
bool    full_antipodal
bool    half_antipodal